### ISSUE TITLE
replacing ut2004 weps?

### ISSUE THREAD

**Issue by lemonjezus** [GreatEmerald/UT3Common#2]
i added all the files to mke the weapon work but when i grab a weapon pack it still gives me ut2004 weapons also. Is there a way to completely replace the default ut2004 weps for ut3?

**Comment by GreatEmerald** [comment]
That is what the included mutator does. Maybe you are using some incompatible additional mutators that manage to work around mine to keep the old weapons.

**Comment by lemonjezus** [comment]
yes i saw no worries! by the way it seems the link gun you made is not able to link like ut2004 does. is there a way for you to change that?

On Dec 18, 2023, at 3:11 AM, Dainius Masiliūnas ***@***.***> wrote:

﻿

That is what the included mutator does. Maybe you are using some incompatible additional mutators that manage to work around mine to keep the old weapons.

**Comment by GreatEmerald** [comment]
It links the way it does in UT3 rather than the way it does in UT2004. There is just no visual effect for it at the moment.

**Comment by lemonjezus** [comment]
ooooh so it does link! thats nice. would you let me know when u add the effects? i rlly like the gun pack you got here i wanna implement it in my server :). also for the announcements you get after a certain number of kills, would there be a way for you to make a config file (ini) to like make those announcements repeat? theyre cool!

On Dec 18, 2023, at 4:38 AM, Dainius Masiliūnas ***@***.***> wrote:

﻿

It links the way it does in UT3 rather than the way it does in UT2004. There is just no visual effect for it at the moment.

**Comment by GreatEmerald** [comment]
Not any time soon, I'm afraid, as I'm busy finishing my PhD. But the code is open, feel free to propose changes :)

**Comment by lemonjezus** [comment]
no problem i could leave a note for that yeah

On Dec 18, 2023, at 2:23 PM, Dainius Masiliūnas ***@***.***> wrote:

﻿

Not any time soon, I'm afraid, as I'm busy finishing my PhD. But the code is open, feel free to propose changes :)
